Source/WebCore/animation/CSSAnimation.cpp

```diff
@@ -134,6 +134,9 @@ void CSSAnimation::syncPropertiesWithBackingAnimation()
         );
     }
 
+    if (!m_overriddenProperties.contains(Property::Range))
+        setRange(animation.range());
```

Comment:
This is good for now, but I think we may have to be able to differentiate between juste `rangeStart` or `rangeEnd` having been set via the JS API and let the other property be set via CSS. When we do, we should add test coverage for that if that isn't already the case.